daemon/logger/factory.go

```diff
@@ -8,13 +8,13 @@ import (
 	"time"
 )
 
-// Creator is a method that builds a logging driver instance with given context
+// Creator is a method that builds a logging driver instance with given context.
```

Comment:
I think it'll be more consistent with stdlib documentation to write just `Creator builds...`, and below too. Also it isn't method at all :)